manual/src/allfiles.etex

```diff
@@ -23,17 +23,16 @@
 \setcounter{page}{1}
 
 \begin{htmlonly}
-\begin{quote}
-\rule{}{}
+\begin{maintitle}
+\vspace*{2ex}
 This manual is also available in
 \ahref{https://ocaml.org/releases/\ocamlversion/ocaml-\ocamlversion-refman.pdf}{PDF},
 \ahref{https://ocaml.org/releases/\ocamlversion/ocaml-\ocamlversion-refman.txt}{plain text},
 as a
 \ahref{https://ocaml.org/releases/\ocamlversion/ocaml-\ocamlversion-refman-html.tar.gz}{bundle of HTML files},
 and as a
 \ahref{https://ocaml.org/releases/\ocamlversion/ocaml-\ocamlversion-refman.info.tar.gz}{bundle of Emacs Info files}.
-\rule{}{}
-\end{quote}
+\end{maintitle}
```

Comment:
This is a hevea 2.35 change, isn't it?

Reply:
It is, before we had horizontal rules and no warning. Losing the rule does not look to be such a problem

Reply:
The maintitle rendering does seem fine to me. In term of structure, having two `div.maintitle`s seems not completely ideal. However, we can improve (or not) that point later and the postprocessed manual is unaffected.